examples/redis/firewallRule.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: cache.azure.jet.crossplane.io/v1alpha1
-kind: FirewallRule
+apiVersion: cache.azure.jet.crossplane.io/v1alpha2
+kind: RedisFirewallRule
```

Comment:
Hi @sergenyalcin,
Did you try applying all of the example manifests (even though may be not waiting for them to become ready) to run `kubectl` validations on them?

Reply:
Hi @ulucinar, yes I tried applying all of the example manifests to run kubectl validations on them. They were successfully applied in cluster.